Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py

```diff
@@ -78,7 +78,8 @@ def CreateOutputFile(self):
             final_crc = hex(self.CalcTiTxtCrc16(self.section_data_index[i]))
 
             #Script Format
-            script_index_crc = "CRC_CHECK "+"0x"+str(final_addr)+' '+str(final_len)+' '+str(final_crc)
+            script_index_crc = "CRC_CHECK 0x{} {} {}".format(
```

Comment:
I just testing @reillyeon branch and it fails. It hang's up. Master works fine.

![image](https://cloud.githubusercontent.com/assets/339144/24739001/6880b27c-1a4e-11e7-9178-238003794fce.png)